.github/workflows/main.yml

```diff
@@ -0,0 +1,60 @@
+name: Main
+
+on:
+  push:
+    branches: [ main, develop, wip ]
+  pull_request:
+    branches: [ main, develop ]
+  schedule:
+    # Every twelve (12) hours.
+    # @see https://crontab.guru/every-12-hours
+    - cron: 0 */12 * * *
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: "${{ matrix.os }}-latest"
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ ubuntu ]
+        php: [ 8.3 ]
```

Comment:
Can we also test on PHP 8.2 and 8.1?

Reply:
Hmm. Per the notes on the PR description above, I omitted them because Drupal 11 has a minimum requirement of 8.3. But now that I look at our `composer.json`, I see we're targeting Drupal _10_. My mistake. Let me fix that.

Reply:
Incidentally, I knew I wasn't crazy for thinking we were targeting Drupal 10... `composer create-project` is creating a Drupal 11 codebase. Now to figure out why. 🤔